L1Trigger/L1TTrackMatch/plugins/L1TrackJetClustering.h

```diff
@@ -17,21 +17,21 @@ namespace l1ttrackjet {
   const unsigned int ETA_INTPART_BITS{3};
   const unsigned int kExtraGlobalPhiBit{4};
 
-  static constexpr int kEtaWordLength=15;
-  static constexpr int kPhiWordLength=12;
+  static constexpr int kEtaWordLength = 15;
+  static constexpr int kPhiWordLength = 12;
 
   //Constants used for jet clustering in eta direction
-  static constexpr int kThirteenBitMask=8191; // 0001111111111111 (bin) = 8191 (dec)
-  static constexpr int kEtaFineBinEdge1=1638;
-  static constexpr int kEtaFineBinEdge2=3276;
-  static constexpr int kEtaFineBinEdge3=4914;
-  static constexpr int kEtaFineBinEdge4=6552;
+  static constexpr int kThirteenBitMask = 8191;  // 0001111111111111 (bin) = 8191 (dec)
+  static constexpr int kEtaFineBinEdge1 = 1638;
+  static constexpr int kEtaFineBinEdge2 = 3276;
+  static constexpr int kEtaFineBinEdge3 = 4914;
+  static constexpr int kEtaFineBinEdge4 = 6552;
```

Comment:
This file I didnt actually touched or developed. what *I think* happened is that when I run the code formats this changed as well. I am saying this because I dont see any change in the algorithm. This values were not set by me, but by the last person modified the code

Reply:
If this isn't yours, don't worry about it.